src/test/java/com/spotify/autoscaler/api/ApiTestResources.java

```diff
@@ -39,6 +39,7 @@
 
   String INSTANCES = "/instances";
   String HEALTH = "/health";
+  String LOAD = "/instances/load";
```

Comment:
i think we should change both this and line 40 to `clusters` instead of `instances`.
both endpoints are accepted but we were supposed to delete `instances` altogether as everything is based on cluster.